### PR TITLE
Add Internationalization documentation 📝

### LINKED ISSUE #3220: FHIR Core Multi Language Support documentation
**Describe the Issue**
We need to document internationalization as used on FHIR Core.

Artifacts that require translations are
- App configs
- Content configs
- Android views

### PR DESCRIPTION
**Pull request by ndegwamartin**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #3220

**Engineer Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 
- [x] I have checked that this PR does NOT introduce **breaking changes** that require an update to **_Content_** and/or **_Configs_**? _If it does add a sample here or a link to exactly what changes need to be made to the content._

**Code Reviewer Checklist**
- [ ] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [ ] I have verified any strings visible on UI components are in the `strings.xml` file
- [ ] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [ ] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]
@ndegwamartin  the documentation looks correct but it seems the configs translation key->value extraction doesn't seem to work well. This [PR](https://github.com/onaio/fhir-resources/pull/3148) has it the issue

File: android/quest/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
     <application
         android:name=".QuestApplication"
-        android:allowBackup="true"
```
[the remark this answers is not in the document]
Comment (ndegwamartin) [reply android/quest/src/main/AndroidManifest.xml:7]: This backs up data to the installer's email

[the remark this answers is not in the document]
Comment (dubdabasoduba) [reply android/quest/src/main/AndroidManifest.xml:7]: Thanks this is helpful https://developer.android.com/guide/topics/data/autobackup

**Review by dubdabasoduba — APPROVED** [review]
@ndegwamartin I am approving this then setting up a call to test the linked PR

**Pull request merged**